pkg/ephemeral/spdz.go

```diff
@@ -124,30 +131,38 @@ func NewSPDZEngine(logger *zap.SugaredLogger, cmder Executor, config *SPDZEngine
 	feeder := NewAmphoraFeeder(logger, config)
 	checker := network.NewTCPChecker(c)
 	proxy := network.NewProxy(logger, config, checker)
+	playerDataPaths, err := preparePlayerData(config)
+	if err != nil {
+		panic(err)
```

Comment:
If the service is not able to prepare the preprocessing data, it generally reaches a state where it cannot recover. In this case the service has to be aborted anyway. 
But I think it would definitely improve code structure and later debugging to panic at the top level.